lib/Doctrine/DBAL/Driver/IBMDB2/DB2Connection.php

```diff
@@ -74,6 +75,7 @@ public function __construct(array $params, $username, $password, $driverOptions
      */
     public function getServerVersion()
     {
+        /** @var stdClass $serverInfo */
         $serverInfo = db2_server_info($this->_conn);
```

Comment:
Does this function always return a `stdClass`? Can't we force it in PHPStan stubs?

Reply:
I didn't actually check whether it is stdClass or not, it returns _some object crate_: http://php.net/db2_server_info

Reply:
oh, and `false`, how unusual ;)